### DISCUSSION TITLE
Reconstruct full point cloud (3D) of real object, from 100 depth frame

### DISCUSSION THREAD

**Discussion by damavand1** [isl-org/Open3D#5813] [General]
hi
first up, I thank you for your efforts to build this valuable library that can improve human life.

I'm new in open 3d and I can't understand your "reconstruction system" yet
http://www.open3d.org/docs/release/tutorial/reconstruction_system/index.html

I have a xbox 360 kinect and I saved depth frame in my custom array
now I want to pass presaved depth frames series saved in memory to some line of codes in python to find matched frames and generate new point cloud of full object  and save it in new a point cloud variable

can you help me ?

**Comment by andreabonvini** [comment]
Hi! I kinda have the same problem, you can check my issue [here](https://github.com/isl-org/Open3D/issues/5804), it would be nice if someone from the Open3D team would answer our questions.

In my case I used the "ground truth" poses extracted from the visualizer, but another option would be to retrieve the poses using the ICP algorithm.

**Reply by damavand1** [reply]
@andreabonvini  yes they dont answer to questions ...
1- andrea, does open3d supported by intel ? you know ?
2- so why they dont suppoert library ?
3- how many project are based on open3D ?
4- does any alternative exists ?
5- does openCV support 3d functions ?
